src/datasets/utils/_hf_hub_fixes.py

```diff
@@ -161,16 +161,18 @@ def list_repo_files(
     repo_id: str,
     revision: Optional[str] = None,
     repo_type: Optional[str] = None,
-    token: Optional[str] = None,
+    use_auth_token: Optional[str] = None,
```

Comment:
"token" has type `Optional[str]` while "use_auth_token" has "Optional[Unnion[bool, str]]", so this is a bit inconsistent. Could you add the support for bool as well here ?

Reply:
You are right: I forgot to update the type hint.